projects/templates/home.html

```diff
@@ -178,8 +178,10 @@ <h2>Culture</h2>
               </a>
               <div>
                 <div class="post-meta"><span class="date">Culture</span> <span class="mx-1">&bullet;</span> <span>Jul 5th '22</span></div>
-                <h3><a href="single-post.html">What is the son of Football Coach John Gruden, Deuce Gruden doing Now?</a></h3>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio placeat exercitationem magni voluptates dolore. Tenetur fugiat voluptates quas, nobis error deserunt aliquam temporibus sapiente, laudantium dolorum itaque libero eos deleniti?</p>
+                {% for culture_posts in culture_posts|slice:"4:5" %}
+                <h3><a href="single-post.html">{{ culture_posts.title }}</a></h3>
+                <p>{{ culture_posts.content|truncatewords:30 }}</p>
+                {% endfor %}
                 <div class="d-flex align-items-center author">
                   <div class="photo"><img src="assets/img/person-2.jpg" alt="" class="img-fluid"></div>
```

Comment:
Images are not loading 
we need to set the static url _**{{ STATIC_URL }}**_ to load the images